sentry/src/test/java/io/sentry/JsonSerializerTest.kt

```diff
@@ -620,7 +620,7 @@ class JsonSerializerTest {
 
         assertEquals("transaction-name", element["transaction"] as String)
         assertEquals("transaction", element["type"] as String)
-        assertNotNull(element["start_timestamp"] as Double)
+        assertNotNull(element["start_timestamp"] as Number)
```

Comment:
This fixes the test but maybe there's actually a problem with the Serializer, not the test itself. Do we want the JSON to not have any decimal places if all places are `0`?

Reply:
Serialized string contains `start_timestamp` that looks like this `1651763016.000000`. So looks like it's only the test, not the production code that's faulty.

Reply:
I'd rather fix the test only then, if the serialization works as intended, all good.

Reply:
Exactly. That's what this line does ;-)